amoro-ams/src/main/java/org/apache/amoro/server/optimizing/maintainer/IcebergTableMaintainer.java

```diff
@@ -321,13 +322,19 @@ protected void cleanContentFiles(
     // so acquire in advance
     // to prevent repeated acquisition
     Set<String> validFiles = orphanFileCleanNeedToExcludeFiles();
-    LOG.info("Starting cleaning orphan content files for table {}", table.name());
+    LOG.info(
+        "Starting cleaning orphan content files for table {} before {}",
+        table.name(),
+        new Timestamp(lastTime));
```

Comment:
Hi, should we use ` java.time.Instant` instead of `java.sql.Timestamp` like:
```
 java.time.Instant..ofEpochMilli(lastTime)
```
which is more general without any dependency rather than JDK.

Reply:
replaced by `java.time.Instant`